pandas/plotting/_misc.py

```diff
@@ -206,7 +206,7 @@ def radviz(frame, class_column, ax=None, color=None, colormap=None, **kwds):
         ...                      'versicolor', 'setosa', 'virginica',
         ...                      'setosa']
         ...     })
-        >>> rad_viz = pd.plotting.radviz(df, 'Category')
+        >>> rad_viz = pd.plotting.radviz(df, 'Category')  # doctest: +SKIP
```

Comment:
But is the plot still shown then in the html documentation?

Reply:
But there are many other plots in the docs? Why only those two? And from reading further it seems you are deactivating matplotlib anyway in the script?

Reply:
I'm not skipping because it's a plot, I'm skipping because it generates a warning, something about the projection (not sure if the warning only happens with the backend `Template`)

Reply:
OK. I don't see any warning locally, though 
(the problem with skipping it in general is that you don't catch typo's or other mistake in this line when using the validation script)